### PR TITLE
ci(e2e): add mainnet manifest

### PR DESCRIPTION
**Pull request by corverroos**
Adds mainnet manifest including keys created by `e2e key-create-all`

issue: #2079

### CODE REVIEW

File: e2e/cmd/keys.go [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+			}
+
+			for _, cfg := range allKeyConfigs(*def) {
+				log.Debug(ctx, "Checking key", "name", cfg.Name, "type", cfg.Type)
+
+				// Check if key already defined
+				if cfg.Type == key.EOA {
+					_, ok := eoa.Address(network, eoa.Role(cfg.Name))
+					if ok {
+						log.Info(ctx, "Skipping eoa key already defined", "name", cfg.Name)
```
Comment (chmllr) [inline e2e/cmd/keys.go:74]: Nit: this is a bit hard to parse without punctuation.
```suggestion
						log.Info(ctx, "Skipping: eoa key already defined", "name", cfg.Name)
```

File: e2e/cmd/keys.go [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+						continue
+					}
+				} else {
+					_, ok := def.Manifest.Keys[cfg.Name][cfg.Type]
+					if ok {
+						log.Info(ctx, "Skipping secret key already in manifest", "name", cfg.Name, "type", cfg.Type)
```
Comment (chmllr) [inline e2e/cmd/keys.go:80]: Same

File: e2e/cmd/keys.go [hunk 3]
```diff
@@ -0,0 +1,252 @@
...
+				Name:    node.Name,
+				Type:    key.Validator,
+			})
+		}
+	}
+
+	return cfgs
+}
+
+// printKeysToml prints the keys to as toml.
```
Comment (chmllr) [inline e2e/cmd/keys.go:224]:
```suggestion
// printKeysToml prints the keys to a writer as toml.
```

**Review by chmllr — APPROVED** [review]

File: e2e/cmd/keys.go [hunk 4]
```diff
@@ -0,0 +1,252 @@
...
+func printKeysToml(keys map[string]map[key.Type]string, w io.Writer) error {
+	keysOnly := struct {
```
Comment (sideninja) [inline e2e/cmd/keys.go:226]: should this write empty map as well? else do a check and return

File: e2e/cmd/keys.go [hunk 5]
```diff
@@ -0,0 +1,252 @@
...
+				}
+
+				if cfg.Type != key.EOA {
+					if _, ok := manifest.Keys[cfg.Name]; !ok {
+						manifest.Keys[cfg.Name] = make(map[key.Type]string)
+					}
+					manifest.Keys[cfg.Name][cfg.Type] = addr
+				}
+			}
+
```
Comment (sideninja) [inline e2e/cmd/keys.go:106]: should we log if no keys were created

File: e2e/cmd/keys.go [hunk 6]
```diff
@@ -0,0 +1,252 @@
...
+	"github.com/omni-network/omni/lib/netconf"
+
+	"github.com/ethereum/go-ethereum/common"
+
+	"github.com/naoina/toml"
+	"github.com/spf13/cobra"
+)
+
+func newKeyCreate(def *app.Definition) *cobra.Command {
+	cfg := key.UploadConfig{}
```
Comment (sideninja) [inline e2e/cmd/keys.go:24]:
```suggestion
	var cfg key.UploadConfig
```

File: e2e/cmd/keys.go [hunk 7]
```diff
@@ -0,0 +1,252 @@
...
+				// Check if key already defined
+				if cfg.Type == key.EOA {
+					_, ok := eoa.Address(network, eoa.Role(cfg.Name))
+					if ok {
+						log.Info(ctx, "Skipping eoa key already defined", "name", cfg.Name)
+						continue
+					}
+				} else {
+					_, ok := def.Manifest.Keys[cfg.Name][cfg.Type]
+					if ok {
```
Comment (sideninja) [inline e2e/cmd/keys.go:79]: nit:
```suggestion
				} else if _, ok := def.Manifest.Keys[cfg.Name][cfg.Type]; ok {
```

File: e2e/cmd/keys.go [hunk 8]
```diff
@@ -0,0 +1,252 @@
...
+// printKeysToml prints the keys to as toml.
+func printKeysToml(keys map[string]map[key.Type]string, w io.Writer) error {
+	keysOnly := struct {
+		Keys map[string]map[key.Type]string `toml:"keys"`
+	}{
+		Keys: keys,
+	}
+	var buf bytes.Buffer
+
+	tomlSettings := toml.Config{
```
Comment (sideninja) [inline e2e/cmd/keys.go:233]: reuse `toml.DefaultConfig` ?

**Review by sideninja — APPROVED** [review]
small comments, but LGTM

File: e2e/cmd/keys.go (hunk 1 above)
Comment (corverroos) [reply e2e/cmd/keys.go:74]: not sure I agree

File: e2e/cmd/keys.go (hunk 4 above)
Comment (corverroos) [reply e2e/cmd/keys.go:226]: this isn't production code, exhaustive validation not super critical

File: e2e/cmd/keys.go (hunk 5 above)
Comment (corverroos) [reply e2e/cmd/keys.go:106]: we already log "skipped"

File: e2e/cmd/keys.go (hunk 6 above)
Comment (corverroos) [reply e2e/cmd/keys.go:24]: I just copied this to another file, didn't change the code, you are welcome to fix thanks

File: e2e/cmd/keys.go (hunk 7 above)
Comment (corverroos) [reply e2e/cmd/keys.go:79]: I like the different if EOA this else that, but sure I see what you mean

File: e2e/cmd/keys.go (hunk 8 above)
Comment (corverroos) [reply e2e/cmd/keys.go:233]: I copied this from e2e/app/geth
